tests/test_micropub.php

```diff
@@ -512,30 +512,6 @@ function test_create_checkin() {
 		$this->assertEquals( '<p>Checked into <a class="h-card p-location" href="http:/a/place">A Place</a>.</p>', $post->post_content );
 	}
 
-	function test_create_checkin_autogenerates_checkin_text_with_content_and_post_kinds() {
```

Comment:
you want to add tests, not remove them. :P we should definitely keep this test and at least use it to check the generated excerpt, if that's where this is instead of content.